packages/perspective-viewer-d3fc/src/js/series/treemap/treemapControls.js

```diff
@@ -0,0 +1,20 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2017, the Perspective Authors.
+ *
+ * This file is part of the Perspective library, distributed under the terms of
+ * the Apache License 2.0.  The full license can be found in the LICENSE file.
+ *
+ */
+
+import {getOrCreateElement} from "../../utils/utils";
+import template from "../../../html/parent-controls.html";
+
+export const getGoToParentControls = container =>
```

Comment:
I think if you have a separate component for the controls, it would be nice to make it a proper component - i.e. it should be able to render the `data.name` itself, and have an `onClick` property that calls a callback function when the button is clicked.

Reply:
Great idea.